tests/monitoring/monitoring.go

```diff
@@ -88,34 +117,71 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 		}, 30*time.Second, 1*time.Second).Should(BeNil())
 	}
 
+	checkAlert := func(alertName string) error {
+		alerts, err := getAlerts(virtClient)
+		if err != nil {
+			return err
+		}
+		for _, alert := range alerts {
+			if string(alert.Labels["alertname"]) == alertName {
+				return nil
+			}
+		}
+		return fmt.Errorf("alert doesn't exist: %v", alertName)
+	}
+
 	verifyAlertExist := func(alertName string) {
 		Eventually(func() error {
-			alerts, err := getAlerts(virtClient)
-			if err != nil {
-				return err
-			}
-			for _, alert := range alerts {
-				if string(alert.Labels["alertname"]) == alertName {
-					return nil
-				}
-			}
-			return fmt.Errorf("alert doesn't exist: %v", alertName)
-		}, 120*time.Second, 1*time.Second).Should(BeNil())
+			return checkAlert(alertName)
+		}, 5*time.Minute, 1*time.Second).Should(BeNil())
+	}
 
+	contains := func(s []string, e string) bool {
+		for _, a := range s {
+			if a == e {
+				return true
+			}
+		}
+		return false
 	}
 
 	waitUntilThereIsNoAlert := func() {
+		ignoredAlerts := []string{"VirtApiRESTErrorsHigh", "VirtControllerRESTErrorsHigh", "VirtOperatorRESTErrorsHigh", "VirtHandlerRESTErrorsHigh"}
+
 		Eventually(func() error {
 			alerts, err := getAlerts(virtClient)
 			if err != nil {
 				return err
 			}
-			if len(alerts) == 0 {
+			numAlerts := len(alerts)
+			for _, alert := range alerts {
+				if contains(ignoredAlerts, string(alert.Labels["alertname"])) {
+					numAlerts--
+				}
+			}
+			if numAlerts == 0 {
 				return nil
 			}
 			return fmt.Errorf("some alerts exist: %v", alerts)
-		}, 120*time.Second, 1*time.Second).Should(BeNil())
+		}, 10*time.Minute, 1*time.Second).Should(BeNil())
```

Comment:
Since the "Virt...RESTErrorsBurst" alerts use historic data they stay firing for longer periods than other alerts even after the problem is solved. They usually take around 5-7 minutes to go back to normal, so I used the next "round" number.